serverless/sql-databases/how-to/connect-to-a-database.mdx

```diff
@@ -15,6 +15,28 @@ categories:
 
 This page shows you how to set up the connection to a Serverless SQL Database using the Scaleway console.
 
+## Overview
+
+To connect to a Serverless SQL Database, you can either use a **connection string**, or **connection parameters**.
+
+  - A connection string provides the necessary information and parameters to establish a connection between an IAM principal (i.e. a user or an application) and the database. The string is written as follows:
```

Comment:
```suggestion
  - A connection string provides the necessary information and parameters to establish a connection between an [IAM User](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/#user) or [Application](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/#application) and the database. The string is written as follows:
```
IAM Principal includes also IAM groups, which cannot cannot to a database. Hence I recall we settled on the standard formulation "IAM User or Application"

Reply:
:+1: but user and application should not be capitalized (to stay consistent with existing doc)